### PR TITLE
ENTESB-19366 Create jira sink kamelets

### PR DESCRIPTION
**Pull request by claudio4j**
* Add a jira issue.
* Add comments to an existing issues in jira
* Update fields on an existing issue in jira
* Transition the status of a jira issue

https://issues.redhat.com/browse/ENTESB-19366

### CODE REVIEW

**Comment by claudio4j** [comment]
@squakez for review.
[ENTESB-19377](https://issues.redhat.com/browse/ENTESB-19377) related to reuse of the icons.

File: library/camel-kamelets-catalog/src/main/resources/kamelets/jira-add-comment-sink.kamelet.yaml [hunk 1]
```diff
@@ -0,0 +1,74 @@
+apiVersion: camel.apache.org/v1alpha1
+kind: Kamelet
+metadata:
+  name: jira-add-comment-sink
+  annotations:
+    camel.apache.org/catalog.version: "kamelet-catalog-1.9-SNAPSHOT"
```
Comment (squakez) [inline library/camel-kamelets-catalog/src/main/resources/kamelets/jira-add-comment-sink.kamelet.yaml]: 1.8

File: library/camel-kamelets-catalog/src/main/resources/kamelets/jira-add-comment-sink.kamelet.yaml [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        - urn:camel:group:credentials
+  types:
+    in:
+      mediaType: application/json
+  dependencies:
+  - "camel:core"
+  - "camel:jackson"
+  - "camel:jira"
+  - "camel:kamelet"
+  - "mvn:com.fasterxml.jackson.datatype:jackson-datatype-joda:2.12.5"
```
Comment (squakez) [inline library/camel-kamelets-catalog/src/main/resources/kamelets/jira-add-comment-sink.kamelet.yaml:57]: Is there any productized one for this?

File: library/camel-kamelets-catalog/src/main/resources/kamelets/jira-add-comment-sink.kamelet.yaml (hunk 1 above)
Comment (claudio4j) [reply library/camel-kamelets-catalog/src/main/resources/kamelets/jira-add-comment-sink.kamelet.yaml]: Ah, forgot to push after I changed to 1.8. Thanks.

File: library/camel-kamelets-catalog/src/main/resources/kamelets/jira-add-comment-sink.kamelet.yaml (hunk 2 above)
Comment (claudio4j) [reply library/camel-kamelets-catalog/src/main/resources/kamelets/jira-add-comment-sink.kamelet.yaml:57]: Yes, [there is 2.12.4](https://mvnrepository.com/artifact/com.fasterxml.jackson.datatype/jackson-datatype-joda?repo=redhat-ga) should we use it ?

**Review by squakez — APPROVED** [review]

**Pull request merged**